go.mod

```diff
@@ -31,6 +31,7 @@ require (
 	github.com/rekby/gpt v0.0.0-20200219180433-a930afbc6edc
 	github.com/safchain/ethtool v0.0.0-20200218184317-f459e2d13664
 	github.com/spf13/pflag v1.0.5
+	github.com/therootcompany/xz v1.0.1
```

Comment:
any reason for using this forked version, than the original https://github.com/xi2/xz ?

Reply:
Licensing. https://github.com/xi2/xz/blob/master/LICENSE doesn't mention an explicit license but https://github.com/therootcompany/xz/blob/master/LICENSE uses CC0 1.0 which is good enough for company's repository system to be happy with pulling it in.